assignment.js

```diff
@@ -65,6 +72,15 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 2:')
 
 // Add your code below this line
+const array = [3, 48, 29, 12, 18]
+let currentLargestNumber = 0
+
+  for(i = 0; i < array.length; i++) {
```

Comment:
@clo-bot  nice use of a `for` loop here. Also good use of using your knowledge of arrays to use `i` as an index to implement the logic of keeping track of the largest number.